src/core_write.cpp

```diff
@@ -223,6 +223,28 @@ void TxToUniv(const CTransaction& tx, const uint256& hashBlock, UniValue& entry,
             const CTxOut& prev_txout = txundo->vprevout[i].out;
             amt_total_in += prev_txout.nValue;
         }
+        if (nullptr != txundo) {
+            switch (verbosity) {
+                case TxVerbosity::SHOW_TXID:
+                case TxVerbosity::SHOW_DETAILS:
+                    break;
+
+                case TxVerbosity::SHOW_DETAILS_AND_PREV_OUT:
+                    const Coin&   prev_coin  = txundo->vprevout[i];
+                    const CTxOut& prev_txout = prev_coin.out;
```

Comment:
We already have `prev_txout` above. Move `prev_coin` up there?

Reply:
Done

Reply:
This is still here...

Reply:
Is this still relevant?